PowerEditor/installer/nativeLang/french.xml

```diff
@@ -21,16 +21,16 @@ Translation note:
 					<Item menuId="macro" name="&amp;Macro"/>
 					<Item menuId="run" name="E&amp;xécution"/>
 					<Item menuId="Plugins" name="Modules d’ex&amp;tension"/>
-					<Item menuId="Window" name="&amp;Documents"/>
+					<Item menuId="Window" name="&amp;Onglets ouverts"/>
```

Comment:
> It's a correct description. However, "Document" is also correct. Both are correct, then let's choose the shorter one. Please keep "Document" unchanged.

I agree with the wording "Documents" that has been used since the beginning on Notepad++ even if "Open tabs" seemed more meaningful to a neophyte.

Reply:
> even if "Open tabs" seemed more meaningful to a average people.

"Window" is a "good old day" menu which was always in the [Multiple Document Interface application](https://learn.microsoft.com/fr-fr/dotnet/desktop/winforms/advanced/multiple-document-interface-mdi-applications?view=netframeworkdesktop-4.8) (Yes, some old guys can't live without it). So for me, it shouldn't be translated as "Onglets" for sure. It might be translated as "Fenêtre" - you might want to check the French term in these MDI applications.